sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -6,10 +6,21 @@
 
 - Supported `withCopyStart` method in `Snapshot` for copying incremental snapshot from incremental snapshot.
 - Supported `awaitCopyStartCompletion` and `awaitCopyStartCompletionAsync` method in `Snapshot`.
+- Supported `copyCompletionPercent` and `copyCompletionError` method in `Snapshot` for retrieving `CopyStart` progress.
 - Supported `copyCompletionPercent` and `copyCompletionError` method in `Snapshot` for retrieving `CopyStart` progress. 
```

Comment:
These seem to be duplicate.

Reply:
thanks. caused by conflict resolve. solved.